compliance/azure/azure_policy_audit/README.md

```diff
@@ -0,0 +1,47 @@
+# Azure: Policy Audit
+
+## What it does
+
+This Policy Template accepts an input that defines which Azure Policies are to be audited.
+
+## Functional Details
+
+- The policy leverages the Azure API to check for assigned policies in all subscriptions the service principal has access to.
+- Assigned policies are compared against the list of policies provided.
+- An email is sent with details on which policies are assigned and not assigned to each subscription.
+
+## Input Parameters
+
+This policy has the following input parameters required when launching the policy.
+
+- *Policy Names* - A list of Azure Policies. **Case sensitive.**
```

Comment:
Does it need to be case sensitive?  I suggest converting to lowercase both values when evaluating.

Reply:
Updated find logic to compare lower case values.